cmd/appgw-ingress/main.go

```diff
@@ -134,7 +134,9 @@ func main() {
 		azClient.SetAuthorizer(authorizer)
 	}
 
-	if _, err = azClient.GetGateway(); err != nil {
+	// check if application gateway exists/have get access
```

Comment:
Nit: can we elaborate on comment, who needs to have access

Reply:
done